jdaviz/app.py

```diff
@@ -2287,7 +2291,7 @@ def vue_data_item_remove(self, event):
         data_label = event['item_name']
         data = self.data_collection[data_label]
         orientation_plugin = self._jdaviz_helper.plugins.get("Orientation")
-        if orientation_plugin is not None:
+        if orientation_plugin is not None and orientation_plugin.align_by == "WCS":
```

Comment:
@bmorris3 - can you confirm if this is correct?  I ran into this bug when adding test coverage (where it would complain otherwise that `data_label == ''` was not available as `new_parent`).  Do you think this should be split out to a separate PR and backported?

Reply:
Yes, I actually just encountered this yesterday too! I was experimenting with @eteq's unloading-data questions and was trying to untangle the bug. It looks like this is probably what we need. Testing this out now.